neuraxle/metaopt/trial.py

```diff
@@ -267,6 +282,7 @@ def __init__(
             start_time: datetime.datetime = None,
             end_time: datetime.datetime = None,
             pipeline: BaseStep = None,
+            should_delete_pipeline_on_completion: bool = True
```

Comment:
Wondering why it's named "should". Does it do it?
```suggestion
            delete_pipeline_on_completion: bool = True
```